app/services/allocations/create_in_nomis.rb

```diff
@@ -0,0 +1,28 @@
+module Allocations
+  class CreateInNomis
+    def self.call(move)
+      booking_id = move.person.latest_nomis_booking_id
+
+      body = {
+        booking_id: booking_id,
+        body_params: {
+          fromPrisonLocation: move.from_location.nomis_agency_id,
```

Comment:
could we ever have an empty nomis agency id?

Reply:
No, the location data is all imported from Nomis and is returned from the Nomis API with an id (primary key) for each location.